README.md

```diff
@@ -115,24 +97,27 @@ Use `in-toto-verify` on the final product to verify that
 - materials and products of each step were in place as defined by the rules, and
 - run the defined inspections
 
-```shell
-in-toto-verify --layout <layout path>
-               {--layout-keys <filepath>[ <filepath> ...],  --gpg <keyid> [ <keyid> ...]}
-               [--gpg-home <path to gpg keyring>]
-               [--verbose]
-```
+For a detailed list of all command line arguments and their usage, run 
+`in-toto-verify --help` or look at the code documentation 
+[here](https://github.com/in-toto/in-toto/blob/develop/in_toto/in_toto_verify.py).
 
 #### Signatures
-`in-toto-sign`, unlike `in-toto-verify`, can only add, replace, and verify signatures within in-toto Link or Layout metadata, with options to
-- replace (default) or add signature(s), with layout metadata able to be signed by multiple keys at once while link metadata can only be signed by one key at a time
-- write signed metadata to a specified path (if no output path is specified, layout metadata is written to the path of the input file while link metadata is written to '<name>.<keyid prefix>.link')
+`in-toto-sign`, unlike `in-toto-verify`, can only add, replace, and 
+verify signatures within in-toto Link or Layout metadata, with options to
+- replace (default) or add signature(s), with layout metadata able to be 
+signed by multiple keys at once while link metadata can only be signed by one key at a time
+- write signed metadata to a specified path (if no output path is specified, 
+layout metadata is written to the path of the input file while link metadata 
+is written to '<name>.<keyid prefix>.link')
```

Comment:
Please use `&lt;` and `&gt` for  &lt;  and &gt; or put the whole filename inside backticks.